### DISCUSSION TITLE
CMD flags documentation

### DISCUSSION THREAD

**Discussion by gabrielottj2** [lllyasviel/Fooocus#3068] [Q&A] [answered]
Guys,

Is there a documentation for all the CMD flags, explaining what each one does?

Thanks.

**Comment by mashb1t** [comment, marked as answer]
Yes, not in a Wiki page, but the code. Please find the notes in https://github.com/lllyasviel/Fooocus/blob/07c6c89edf34676da86310dc1749db8ce3d082af/args_manager.py and https://github.com/lllyasviel/Fooocus/blob/07c6c89edf34676da86310dc1749db8ce3d082af/ldm_patched/modules/args_parser.py.

**Reply by gabrielottj2** [reply]
Thanks!
